pkgs/tools/inputmethods/gebaar-libinput/default.nix

```diff
@@ -0,0 +1,29 @@
+{ gcc8Stdenv, fetchFromGitHub
```

Comment:
Usually, we import in the `default.nix` file `stdenv` and then `callPackage` with `stdenv = gcc8Stdenv`.